stpsf/roman.py

```diff
@@ -271,8 +288,13 @@ def __init__(self, *args, **kwargs):
         self.siaf = stpsf_core.get_siaf_with_caching('roman')
         self._aperturename = None
 
+        # reassign self.options from dict to _RomanInstrumentOptionsDict,
+        # mainly to prevent user modification of the add_distortion key
+        self.options['add_distortion'] = 'NA'  # distortion is always on
+        self.options = _RomanInstrumentOptionsDict(self.options.copy())
+
         self.options['jitter'] = 'gaussian'
-        self.options['jitter_sigma'] = 0.012  # arcsec/axis, see https://roman.ipac.caltech.edu/sims/Param_db.html#telescope
+        self.options['jitter_sigma'] = 0.012  # arcsec/axis, see https://roman.ipac.caltech.edu/page/param-db#telescope
```

Comment:
Let's point to the GSFC Roman Technical Information repo instead: https://github.com/RomanSpaceTelescope/roman-technical-information/tree/main/data/Observatory/MissionandObservatoryTechnicalOverview